### PR TITLE
Callback action v2

### PR DESCRIPTION
**Pull request by hasinaxp**
callback action upgrades -> standalone, shorten url, state share between pyscript
testcase fixes and new test cases
prepare_request_with_bot

## Summary by CodeRabbit

- **New Features**
	- Enhanced callback action functionality with improved handling of identifiers and callback URLs.
	- Added new fields to callback configurations for flexible management, including encryption options.
	- Introduced error handling improvements and a new request preparation method for asynchronous callbacks.
	- Updated routing structure for a more streamlined process with token-based identification.
	- Implemented security enhancements for handling sensitive data through encryption methods.
	- Added a new API endpoint to retrieve standalone callback URLs.

- **Bug Fixes**
	- Improved logging clarity in response handling and test outputs.

- **Tests**
	- Expanded test cases for callback configurations and added new scenarios for validation.
	- Updated existing tests to align with recent changes in callback processing logic.
	- Enhanced tests for encryption and decryption functionalities to ensure security measures are effective.

### CODE REVIEW

File: kairon/api/models.py [hunk 1]
```diff
@@ -1292,6 +1292,10 @@ class CallbackConfigRequest(BaseModel):
     pyscript_code: str
     validation_secret: str = None
     execution_mode: str = CallbackExecutionMode.ASYNC.value
+    standalone: bool = False
+    shorten_token: bool = False
+    standalone_id_path: str = ''
```
Comment (sfahad1414) [inline kairon/api/models.py:1297]: Make standalone_id_path as None

**Review by sfahad1414 — CHANGES_REQUESTED** [review]
do the change

**Review by hiteshghuge — APPROVED** [review]
review

**Review by hiteshghuge — APPROVED** [review]
approved

**Review by sfahad1414 — APPROVED** [review]

**Pull request merged**